pages/querying/vector-search.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Vector search
+description: Learn how to use vector search and manage vector indices in Memgraph.
+---
+
+import { Callout } from 'nextra/components'
+
+# Vector search
+
+The vector search is currently an experimental feature. For the exact status
+please visit the [experimental
+page](/database-management/experimental-features). To enable the feature use
+the following `--experimental-enabled=vector-search`. To configure the feature
+use `--experimental-config`. For example:
+```shell
+--experimental-config=
+'
+{
+  "vector-search": {
+    "index_name": {
+      "label": "Node",
+      "property": "vector",
+      "dimension": 2,
+      "capacity": 1000,
+      "metric": "cos",
+      "scalar": "f32",
+      "resize_coefficient": 2,
+    }
+  }
+}
+'
+```
+Keep in mind that both flags are required and the configuration requires the
+following mandatory fields: `label`, `property`, `dimension` and `capacity`.
+
+### Metric Kinds
+
+The following table lists the supported metric kinds for vector search. These
+metrics determine how distances between vectors are calculated. Default type
+for the metric is `l2sq`.
+
+| Metric      | Description                                          |
+|-------------|------------------------------------------------------|
+| `ip`        | Inner product (dot product).                         |
+| `cos`       | Cosine similarity.                                   |
+| `l2sq`      | Squared Euclidean distance.                          |
+| `pearson`   | Pearson correlation coefficient.                     |
+| `haversine` | Haversine distance (suitable for geographic data).   |
+| `divergence`| A divergence-based metric.                           |
+| `hamming`   | Hamming distance.                                    |
+| `tanimoto`  | Tanimoto coefficient.                                |
+| `sorensen`  | Sørensen-Dice coefficient.                           |
+| `jaccard`   | Jaccard index.                                       |
+
+### Scalar Kinds
+
+Scalar kinds define the data type of each vector element. Default type for the
+metric is `f32`.
```

Comment:
Let's sync to determine if it makes sense to include all these scalar types in the documentation, since we are casting everything to a float at the end.

Reply:
Just put a note about that at the beginning, feel free to just update directly what you think it's the best 💪